django-backend/fecfiler/scha_transactions/migrations/0008_schatransaction_parent_transaction.py

```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.12 on 2022-06-23 13:59
```

Comment:
I'm pretty certain you still need to not have this 0008 migration because matt introduced an 0008 in his recent change

Reply:
:confetti_ball: Done!  Thanks for the help! :confetti_ball: